### PR TITLE
Feat: Cards

### PR DESCRIPTION
**Pull request by NandhiniKrishnamoorthy**
Please review card component.
You can have a preview here: https://card-componentui.netlify.app/

### CODE REVIEW

File: css/badge.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    line-height: 1;
+    position: absolute;
+    left: 70%;
+    top: 0;
+    transform: translate(-50%, -50%);
+    white-space: nowrap;
+}
+
+.badge-red{
+    background-color: red;
```
Comment (Naveenchandar) [inline css/badge.css:24]: Please use hex codes for consistency and if the same color is repeated, declare a variable and use it.

File: css/badge.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+.badge-red-avatar:hover + .badge_description,
+.badge-yellow-avatar:hover + .badge_description,
+.badge-grey-avatar:hover + .badge_description {
+    display: block;
+    background-color: black;
+    color: white;
+    position: absolute;
+    left: 72%;
+    top: 50%;
+}
```
Comment (Naveenchandar) [inline css/badge.css:57]: If these are common properties for all the classes, You can use a common class for all these badges.

File: css/card.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    cursor: pointer;
+    background-color: black;
+    color: var(--golden-color);
+    width: 300px;
+    height: max-content;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    margin: 1rem 1rem 1rem 1rem;
```
Comment (Naveenchandar) [inline css/card.css:11]: Please use shorthand for margin.

File: css/card.css [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+.card_main:hover .text1-on-card,
+.card_03:hover .text1-on-card{    
+    display: block;
+    background-color: var(--golden-color);
+    color: black;
+    position: absolute;
+    left: 40%;
+    top: 45%;
+    padding: 7px 15px 7px 15px;
```
Comment (Naveenchandar) [inline css/card.css:36]: Please use shorthand for padding.

File: css/card.css [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+.card_02 {
+    background-color: black;
+    color: var(--golden-color);
+    width: 300px;
+    height: max-content;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    margin: 1rem 1rem 1rem 1rem;
```
Comment (Naveenchandar) [inline css/card.css:48]: same styles repeated in **.card_main**. Please use the same class.

File: css/card.css [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.card-overlay {
+    inset: 0;
+    color: white;
+    background-color: var(--overlay);
+    position: absolute;   
+    padding: 7px 15px 7px 15px;
```
Comment (Naveenchandar) [inline css/card.css:56]: Please use shorthand for padding.

File: css/card.css [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+.card-price-discount{
+    text-decoration: line-through;
+    font-size: 0.75rem;
+    padding-left: 4px;
+}
+
+.card-deal{
+    margin-top: 1rem;
+    margin-bottom: 0;
```
Comment (Naveenchandar) [inline css/card.css:71]: Please use shorthand for margin.

File: css/card.css [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+    border: lightgrey 1px solid;
+    color: lightgrey;
+    padding: 5px 5px 5px 5px;
```
Comment (Naveenchandar) [inline css/card.css:74]: Please use shorthand for padding.

File: css/card.css [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.card_03 {
+    background-color: black;
+    width: 400px;
+    height: 230px;
```
Comment (Naveenchandar) [inline css/card.css:80]: Please use relative units.

File: css/card.css [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+    position: absolute;
+}
+
+.card_text_only{
+    width: 250px;
+    height: max-content;
+    background-color: black;
+    color: white;
+    margin: auto;
+    padding: 15px 15px 15px 15px;
```
Comment (Naveenchandar) [inline css/card.css:112]: Please use shorthand for padding.

File: css/card.css [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+.card-textonly-desc {
+    padding: 5px 5px 5px 5px;
+    text-align: justify;
+    line-height: 1.30rem;
+}
+
+.card-textonly-button {
+    width: 100%;
+    margin: 25px 0px 0px 0px;
+    padding: 5px 0px 5px 0px;
```
Comment (Naveenchandar) [inline css/card.css:130]: Please use shorthand.

File: index.html [hunk 12]
```diff
@@ -14,7 +14,35 @@
     <header class="header m-auto py-0p5 pl-2">
         <h2>Components</h2>
     </header>
-    <container>        
+    <container>
```
Comment (Naveenchandar) [inline index.html:17]: Not sure about this tag in HTML. Please once verify it.

File: css/badge.css [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+    transform: translate(-50%, -50%);
+    white-space: nowrap;
+}
+
+.badge-red{
+    background-color: red;
+}
+
+.badge-green {
+    background-color: #3DC94F;
```
Comment (AtulHande03a) [inline css/badge.css:28]: use variables for color codes

File: css/card.css [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+
+.card-desc-horizontal {
+    color: var(--golden-color);
+    padding-left: 65%;
+    padding-top: 10%;
+    position: absolute;
+}
+
+.card_text_only{
+    width: 250px;
```
Comment (AtulHande03a) [inline css/card.css:107]: use consistent unit ie either rem or px.. recommended rem

**Review by AtulHande03a — APPROVED** [review]
other than given comments, rest all is good

**Review by maitrakhatri — APPROVED** [review]
Consider Naveen's pointed out changes, rest looks good

**Pull request merged**